pandas/core/ops.py

```diff
@@ -293,6 +319,28 @@ def _get_frame_op_default_axis(name):
 Returns
 -------
 result : DataFrame
+
+Examples
+--------
+>>> a = pd.DataFrame([1, 1, 1, np.nan], index=['a', 'b', 'c', 'd'])
+     0
+a  1.0
+b  1.0
+c  1.0
+d  NaN
+>>> b = pd.DataFrame([[1, np.nan], [np.nan, 2], [1, np.nan], [np.nan, 2]], index=['a', 'b', 'c_', 'd'])
```

Comment:
this will not pass the linter, need to format this, prob easier to use a dict to construct as well.